matthenge/weather-app
# Add responsiveness and UI enhancements

#### What does this PR do?

- Add responsiveness and UI enhancements

#### Description of Task to be completed?

- Add responsiveness
- Add UX enhancements
- Add tests for new files

#### How should this be manually tested?

- Clone the repo
- Setup the codebase and run
- The app should be responsive to different screen sizes

#### Any background context you want to provide?

#### What are the relevant issue Ticket?

#### Screenshots (if appropriate)

#### Lighthouse Screenshots (if appropriate)

#### Check List

#### Questions:
